source/daplink/RTX_Config.c

```diff
@@ -90,7 +90,7 @@
 #elif defined(INTERFACE_SAM3U2C)
 #define OS_CLOCK    96000000
 #elif defined(INTERFACE_LPC4322)
-#define OS_CLOCK    204000000
+#define OS_CLOCK    96000000
```

Comment:
Older boards were using 204MHz. Is there a new board that is powering the LPC4322 with a lower voltage? If so can you provide any details on that and the voltage it operates at?